.travis.yml

```diff
@@ -7,14 +7,21 @@ go_import_path: github.com/google/certificate-transparency-go
 
 
 env:
+  global:
+  - GO111MODULE=on
+  matrix:
   - GOFLAGS=
-  - GOFLAGS=-race
+  - GOFLAGS=-race NO_LINTER=true
```

Comment:
Perhaps we only need to only run the linter in the base case? (is there any code which would be linted by `WITH_COVERAGE=true` but not in the empty `GOFLAGS` case?)

Reply:
I can't think of any value added by running the linter with coverage on